tests/phpunit/elementor/modules/atomic-widgets/prop-types/test-linked-dimensions-prop-type.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Elementor\Testing\Modules\AtomicWidgets\PropTypes;
+
+use Elementor\Modules\AtomicWidgets\PropTypes\Linked_Dimensions_Prop_Type;
+use Elementor\Modules\AtomicWidgets\PropTypes\Size_Prop_Type;
+use ElementorEditorTesting\Elementor_Test_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Test_Linked_Dimensions_Prop_Type extends Elementor_Test_Base {
```

Comment:
I think we should avoid doing a lot of unit tests, as it will be much harder to change the code later, try to do "Component" tests which actual test the logic and not specific class.